plugins/repository-hdfs/build.gradle

```diff
@@ -44,7 +44,7 @@ dependencies {
   compile 'org.apache.htrace:htrace-core:3.1.0-incubating'
   compile 'com.google.guava:guava:11.0.2'
   compile 'com.google.protobuf:protobuf-java:2.5.0'
-  compile 'commons-logging:commons-logging:1.1.3'
+  compile 'commons-logging:commons-logging:1.2'
```

Comment:
No we can't do this one, the version of Hadoop here depends on [1.1.3](https://github.com/apache/hadoop/blob/release-2.7.1/hadoop-project/pom.xml#L591-L610).